[IMP] finance/fiscal_localizations: Internal Reverse Charge (#8054)
odoo/documentation: content/applications/finance/fiscal_localizations/italy.rst

   differentiate them easily.

.. _italy/reverse-charge:

Reverse charge
==============

The **reverse charge** mechanism is a VAT rule that shifts the liability to pay VAT from the
supplier to the customer. The customers pay the VAT *themselves* to the :abbr:`AdE (Agenzia delle
Entrate)` instead. There are different types:

- | :guilabel:`Internal Reverse Charge` (for domestic sales)
  | The VAT responsability is shifted to the buyer for certain categories of products and services
    where the risk of tax evasion is higher.
- | :guilabel:`External Reverse Charge` (for intra-EU sales)
  | VAT is due in the country of the delivery or in the country where the service is performed.
    When the buyer is itself an Italian business, then the EU offers a mechanism that allows the
    seller to transfer his responsibility to the buyer.

Invoices
--------

**Reverse-charged** customer invoices show no VAT amount, but the :abbr:`AdE (Agenzia delle
Entrate)` requires the seller to specify the :guilabel:`Tax Exemption` reason and the :guilabel:`Law
Reference` that enable the reverse-charge mechanism. Odoo provides a set of special 0% taxes that

> can be assigned to each reverse-charged invoice lines, with the most commonly used configurations.

Author: This bit was a little messy, so I tried to follow TSB and JCS instructions.

Chiara didn't like the "tax evasion" sentence much, but I'm using it again because otherwise I don't know how to clarify Internal Reverse Charge.

Let me know what you people think, I'm open to suggestions.
Thank you all for the hard work. 🙏🏻

.. _italy/reverse-charge:

Reverse charge
==============

The **reverse charge** mechanism is a VAT rule that shifts the liability to pay VAT from the
supplier to the customer. The customers pay the VAT *themselves* to the :abbr:`AdE (Agenzia delle
Entrate)` instead. There are different types:

- | :guilabel:`Internal Reverse Charge` (for domestic sales)
  | The VAT responsability is shifted to the buyer for certain categories of products and services
    where the risk of tax evasion is higher.
- | :guilabel:`External Reverse Charge` (for intra-EU sales)
  | VAT is due in the country of the delivery or in the country where the service is performed.
    When the buyer is itself an Italian business, then the EU offers a mechanism that allows the
    seller to transfer his responsibility to the buyer.

Invoices
--------

**Reverse-charged** customer invoices show no VAT amount, but the :abbr:`AdE (Agenzia delle
Entrate)` requires the seller to specify the :guilabel:`Tax Exemption` reason and the :guilabel:`Law
Reference` that enable the reverse-charge mechanism. Odoo provides a set of special 0% taxes that
can be assigned to each reverse-charged invoice lines, representing the most

> commonly used configurations.

Author: This bit was a little messy, so I tried to follow TSB and JCS instructions.

Chiara didn't like the "tax evasion" sentence much, but I'm using it again because otherwise I don't know how to clarify Internal Reverse Charge.

Let me know what you people think, I'm open to suggestions.
Thank you all for the hard work. 🙏🏻

   --> Configuration --> Taxes`, select a similar tax, then click on the cog icon and select
   :guilabel:`Duplicate`. In the :guilabel:`Advanced Options` tab, add the :guilabel:`Exoneration`
   and :guilabel:`Law Reference`, to confirm click on :guilabel:`Save`.

.. tip::
   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
   differentiate them easily.

.. _italy/reverse-charge:

Reverse charge
==============

Italian businesses selling goods and services are sometimes required *not* to charge the customer
for the VAT. The customers pay the VAT *themselves* to the :abbr:`AdE (Agenzia delle Entrate)`
instead. This mechanism is called **reverse charge**; there are different types:

- :guilabel:`Internal Reverse Charge` (for domestic sales) is when the VAT is paid by the seller.

- :guilabel:`External Reverse Charge` (for intra-EU sales). VAT is a type of tax that is applied to
  the country of the buyer. The seller has no means to collect and pay VAT in the other country, so
  the buyer is responsible for doing it in a B2B setting.

Invoices
--------

> **Reverse charged** customer invoices are **tax exempt** (0%) for the seller. To use it on an invoice,

Reviewer: **Reverse-charged** customer invoices are **tax-exempt** (0%) for the seller

> **Reverse-charged** customer invoices are **tax exempt** (0%) for the seller. To use it on an invoice,

Reviewer 2: The wording is incorrect. The transaction is not exempt of taxes. The responsibility to declare it is transfered on the buyer.

Vendor bills
------------

Italian companies subjected to Reverse Charge must send the information in the bill received to the
:abbr:`AdE (Agenzia delle Entrate)`.

.. note::
   Self-reported VAT XML files must be issued and sent to the :abbr:`AdE (Agenzia Delle Entrate)`
   for reverse charged bills.

When creating a vendor bill, **reverse charge** taxes are available to be added in the :guilabel:`Taxes`
field. You can check which taxes are available by going to :menuselection:`Accounting -->
Configuration --> Taxes`, you can see that the 10% Goods and 22% Services taxes are activated, among
others. Due to the automated configuration of the Italian fiscal position, these are automatically
activated in the tax list.

.. _italy/grids:

Tax grids
---------

The Italian localization has a specific **tax grid** section for **reverse charge** taxes. These tax
grids are identifiable by the :ref:`VJ <italy/grids>` tag, and can be found by going to
:menuselection:`Accounting --> Reporting --> Audit Reports: Tax Report`.

.. image:: italy/italy-grids.png
   :alt: Reverse charge tax grid from the VJ section of the Tax Declaration

.. _italy/e-invoicing:

E-invoicing
===========

> The :abbr:`SdI (Sistema di Interscambio)` is the electronic invoicing system used in Italy. It
> enables to send and receive electronic invoices to and from customers. The documents must be in an
> XML :abbr:`EDI (Electronic Data Interchange)` format called **FatturaPA** and formally validated
> by the system before being delivered.

Reviewer 3: ```suggestion
The :abbr:`SdI (Sistema di Interscambio)` is the electronic invoicing system used in Italy. It
enables the sending and receiving of electronic invoices to and from customers. The documents must
be in an XML :abbr:`EDI (Electronic Data Interchange)` format called **FatturaPA** and formally
validated by the system before being delivered.
```

   of the following fake :guilabel:`Destination Code` given by the :abbr:`AdE (Agenzia Delle Entrate)`:
   `0803HR0` - `N8MIMM9` - `X9XX79Z`. Any real production :guilabel:`Codice Destinario` of your
   customers will not be recognized as valid by the test service.

.. image:: italy/italy-edi.png
   :alt: Electronic document invoicing settings

.. _italy/e-invoicing-process:

Process
-------

The submission of invoices to the :abbr:`SdI (Sistema di Interscambio)` for Italy is an electronic
process used for the mandatory transmission of tax documents in XML format between companies and
the :abbr:`AdE (Agenzia delle Entrate)` to reduce errors and verify the correctness of operations.

.. note::
   You can check the current status of an invoice by the :guilabel:`SdI State` field.
   The XML file can be found as an **attachment** of the invoice.

.. image:: italy/italy-edi-process.png
   :alt: EDI system architecture

XML Documents creation
~~~~~~~~~~~~~~~~~~~~~~

Odoo generates the required XML files as attachments to invoices in the `FatturaPA` format
required by the :abbr:`AdE (Agenzia delle Entrate)`. Select the invoices and mark

> :guilabel:`Generate XML File` in the :guilabel:`Send and Print` dialog to generate the attachments.

Author: The attachments are not in the tab, they're just attached to the invoice.
I've also specified which selected action does what, both in this paragraph and in the following one.

.. image:: italy/italy-edi-menu.png
   :alt: Send and Print menu

.. image:: italy/italy-edi-send-and-print.png
   :alt: Send and Print dialog

.. image:: italy/italy-edi-attachments.png
   :alt: EDI Attachments

Submission to SDI
~~~~~~~~~~~~~~~~~

Invoices are sent to the :guilabel:`Proxy Server`, which gathers all requests and then forwards them
via a WebServices channel to the :abbr:`SdI (Sistema di Interscambio)`. Check the sending status of
the invoice through the :guilabel:`Check Sending` button at the top of the invoice's view.

Processing by SDI
~~~~~~~~~~~~~~~~~

The :abbr:`SdI (Sistema di Interscambio)` receives the document and verifies for any errors. At this

> stage, the invoice is in the :guilabel:`SdI Processing` state, visible on the invoice's view.

Reviewer: stage, the invoice is in the :guilabel:`SdI Processing` state, as shown on the invoice.

   - :ref:`italy/split-payment`; as a :abbr:`PA (Public Administration)` business buyer, you have to
     pay taxes and integrate tax information. Be sure that you replace the :guilabel:`0% Sale Taxes` on
     the Vendor Bill you received with the correct :guilabel:`Split Payment` taxes.
   - :guilabel:`Self Consumption`; when as a business owner you use an asset that you bought for
     business for personal reasons instead, you have to pay those taxes you originally deducted as
     a business cost for it.

Odoo may detect that your Vendor Bill can be interpreted as a document of a type that needs Tax
Integration, as detailed in the :ref:`italy/document-types` section.

.. important::
   Be sure that you replace the :guilabel:`0% Sale Taxes` on the vendor bill you received with the ones
   you're supposed to pay to the :abbr:`AdE (Agenzia delle Entrate)`. A button then appears on the
   top of the single vendor bill form to send them.

   When clicking on the send tax button, an XML file of the appropriate :guilabel:`Document Type` will
   be generated, attached to the bill and sent as for invoices (put the link of the paragraph you're

>    talking about).

Reviewer: I think the line in brackets shouldn't have been there?

   .. image:: italy/italy-edi-tax-integration-button.png
      :alt: EDI Send Tax Integration button

.. _italy/document-types:

Document Types
--------------

The :abbr:`SdI (Sistema di Interscambio)` requires businesses to send customer invoices and other
documents through the :abbr:`EDI (Electronic Data Interchange)`.
The following :guilabel:`Document Type` codes all technically identify different business use cases.

.. note::
   We do **not** show the :guilabel:`Document Type` to the user, but it is used in the generation of

>    the XML file.

Reviewer: once again, not sure why this is interesting here, we give the reader info that can be useful but this seems more of a technical part behind how it works

Author: See the discussion with Benj on the other PR for the reasons.
Will remove this sentence for now.

   Selecting either :guilabel:`Test (experimental)` or :guilabel:`Official` is **irreversible**.
   Once in :guilabel:`Official` mode, it is not possible to select :guilabel:`Test (experimental)`
   or :guilabel:`Demo`, and same for :guilabel:`Test (experimental)`. We recommend creating a

>    **separate database** for testing purposes only.

Reviewer 3: I suggest this formulation

```
.. warning::
   Selecting either :guilabel:`Test (experimental)` or :guilabel:`Official` is **irreversible**.
   For example, once in :guilabel:`Official` mode, it is not possible to select :guilabel:`Test
   (experimental)` or :guilabel:`Demo`. We recommend creating a **separate database** for testing
   purposes only.
```

.. note::
   When in :guilabel:`Test (Experimental)` mode, all invoices sent *must* have a partner using one
   of the following fake :guilabel:`Destination Code` given by the :abbr:`AdE (Agenzia Delle Entrate)`:
   `0803HR0` - `N8MIMM9` - `X9XX79Z`. Any real production :guilabel:`Codice Destinario` of your
   customers will not be recognized as valid by the test service.

.. image:: italy/italy-edi.png
   :alt: Electronic document invoicing settings

.. _italy/e-invoicing-process:

Process
-------

The submission of invoices to the :abbr:`SdI (Sistema di Interscambio)` for Italy is an electronic
process used for the mandatory transmission of tax documents in XML format between companies and
the :abbr:`AdE (Agenzia delle Entrate)` to reduce errors and verify the correctness of operations.

> .. note::
>    You can check the current status of an invoice by the :guilabel:`SdI State` field.
>    The XML file can be found as an **attachment** of the invoice.

Reviewer 3: ```suggestion
.. note::
   You can check the current status of an invoice by the :guilabel:`SdI State` field. The XML file
   is attached to the invoice.
```

(Sistema di Interscambio)`'s observations will be inserted at the top of the Invoice tab.
To resolve the issue, it is sufficient to delete the attachments of the invoice, return the
invoice to :guilabel:`Draft`, and fix the errors. Once the invoice is ready, it can be resent.

.. note::
   To regenerate the XML, both the XML attachment and the PDF report must be deleted, so
   that they are then regenerated together. This ensures that both always contain the same data.

.. image:: italy/italy-edi-rejected.png
   :alt: EDI Rejected State

Forwarding Completed
~~~~~~~~~~~~~~~~~~~~

The invoice has been delivered to the customer; however, you can still send a copy to the customer
in PDF via email or post. Its status is :guilabel:`Accepted by SDI, Delivered to Partner`.

If the :abbr:`SdI (Sistema di Interscambio)` cannot contact your customer, they may not be
registered on the :abbr:`AdE (Agenzia delle Entrate)` portal. In this case, just make sure to send
the invoice in PDF via email or by mail. The invoice will be in the :guilabel:`Accepted by SDI,
Partner Delivery Failed` state.

Tax Integration
---------------

> When you receive a vendor bill, either from :abbr:`SdI (Sistema di Interscambio)`, from paper or
> from an imported XML file, the Tax Agency might request you to send some tax information integration
> back to the :abbr:`SdI (Sistema di Interscambio)`. It happens when a transaction that was tax exempt
> becomes taxable for any reason.

Reviewer 3: ```suggestion
When you receive a vendor bill, either from :abbr:`SdI (Sistema di Interscambio)`, from paper or
from an imported XML file, the Tax Agency might request that you send some tax information
integration back to the :abbr:`SdI (Sistema di Interscambio)`. It happens when a transaction that
was tax exempt becomes taxable for any reason.
```

.. example::
   Here is a non-exhaustive list:

   - :ref:`italy/reverse-charge`; as a buyer you have to pay taxes on what you buy and integrate
     tax information. :guilabel:`Reverse Charge` taxes.
   - :ref:`italy/split-payment`; as a :abbr:`PA (Public Administration)` business buyer, you have to
     pay taxes and integrate tax information. Be sure that you replace the :guilabel:`0% Sale Taxes` on
     the Vendor Bill you received with the correct :guilabel:`Split Payment` taxes.

>    - :guilabel:`Self Consumption`; when as a business owner you use an asset that you bought for

Reviewer 3: ```suggestion
   - :guilabel:`Self Consumption`; when, as a business owner, you use an asset that you bought for
```

   top of the single vendor bill form to send them.

   When clicking on the :guilabel:`Send Tax Integration` button, an XML file of the appropriate
   :guilabel:`Document Type` will be generated, attached to the bill and sent as for invoices.

   .. image:: italy/italy-edi-tax-integration-button.png
      :alt: EDI Send Tax Integration button

.. _italy/document-types:

Document Types
--------------

The :abbr:`SdI (Sistema di Interscambio)` requires businesses to send customer invoices and other
documents through the :abbr:`EDI (Electronic Data Interchange)`.
The following :guilabel:`Document Type` codes all technically identify different business use cases.

TD01 - Invoices
~~~~~~~~~~~~~~~

This represents the standard **domestic** scenario for all invoices exchanged through the :abbr:`SdI
(Sistema di Interscambio)`. Any invoice that doesn't fall into one of the specific special cases
will be categorized as a Regular Invoice, identified by the :guilabel:`Document Type` `TD01`.

TD02 - Down payments
~~~~~~~~~~~~~~~~~~~~

**Down payment** invoices are imported/exported with a different :guilabel:`Document Type` code

> `TDO2` than regular invoices. Upon import of the invoice, it creates a regular vendor bill.

Reviewer 3: ```suggestion
`TDO2` than regular invoices. Upon import of the invoice, a regular vendor bill is created.
```

field with the special value `TD28`.

Odoo exports a transaction as `TD28` if the following conditions are met:

#. It is a vendor bill;
#. It has at least one tax on the invoice lines targets the tax grids :ref:`VJ <italy/grids>`;
#. The **country** of the partner is **San Marino**.

Public Admnistration Businesses (B2G)
=====================================

:abbr:`PA (Public Administration)` businesses are subjected to more control than private businesses
as they handle public money coming from taxpayers. The :abbr:`EDI (Electronic Data Interchange)`
process adds some steps to the :ref:`regular one <italy/e-invoicing-process>` as :abbr:`PA (Public
Administration)` businesses can **accept** or **refuse** invoices.

.. note::
   :abbr:`PA (Public Administration)` businesses have a 6 digits long :guilabel:`Destination Code`
   also called :abbr:`CUU (Codice Univoco Ufficio)` that is **mandatory**, **PEC** address cannot be
   used in this case.

.. seealso::
   The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
   found here along with their :guilabel:`Destination Code`:

>    - `List of PA businesses with their Destination Code <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_

Reviewer: I'd put the link when you say 'here' 
   The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
   found `here   - `List of PA businesses with their Destination Code <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
 along with their :guilabel:`Destination Code`

When you receive a vendor bill, either from :abbr:`SdI (Sistema di Interscambio)`, from paper or
from an imported XML file, the Tax Agency might request that you send some tax information
integration back to the :abbr:`SdI (Sistema di Interscambio)`. It happens when a transaction that

> was tax)exempt becomes taxable for any reason.

Author: Stray round bracket?

.. seealso::
   The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
   found here - `List of PA businesses <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_

>    along with their Destination Code.

Reviewer: what I meant:
  The complete list of businesses that belong to the :abbr:`PA (Public Administration)` can be
   found  `here <https://www.agenziaentrate.gov.it/portale/web/guest/aree-tematiche/fatturazione-elettronica>`_
   along with their Destination Code.

If the XML file requires it, the :abbr:`AdE (Agenzia Delle Entrate)` can *only* proceed payments
of electronic invoices when the XML file contains a :abbr:`CIG (Codice Identificativo Gara)` and
:abbr:`CUP (Codice Unico di Progetto)`.

.. note::
   - The :abbr:`CUP (Codice Unico di Progetto)` and the :abbr:`CIG (Codice Identificativo Gara)`
     must be included in one of the `DatiOrdineAcquisto`, `DatiContratto`, `DatiConvenzione`,
     `DateRicezione`, or `DatiFattureCollegate` XML tags.
     These correspond to the elements named :guilabel:`CodiceCUP` and :guilabel:`CodiceCIG` of the
     electronic invoice XML file, whose table can be found on the government `website
     <http://www.fatturapa.gov.it/>`_.

.. _italy/split-payment:

Split Payment
-------------

The :guilabel:`Split Payment` mechanism behaves much like :ref:`italy/reverse-charge`.

.. example::
   When an Italian company bills a :abbr:`PA (Public Administration)` business, for example they're
   selling cleaning services for a public building, the :abbr:`PA (Public Administration)` business
   will self-report the VAT to the Tax Agency themselves, and the vendor just has to select the

>    appropriate tax with the right :guilabel:`Tax Exemption` for their invoice lines.

Reviewer 3: ```suggestion
   When an Italian company bills a :abbr:`PA (Public Administration)` business - for example,
   cleaning services for a public building - the :abbr:`PA (Public Administration)` business
   self-reports the VAT to the Tax Agency themselves, and the vendor just has to select the
   appropriate tax with the right :guilabel:`Tax Exemption` for their invoice lines.
```

   Destinatario` is detected, then the :abbr:`EDI (Electronic Data Interchange)` process stops, and
   the invoice is set on the :guilabel:`Requires user signature` state. You can download the
   document in XML, sign it with any :guilabel:`Digital Qualified Signature` provider's external
   program and send it through the :abbr:`AdE (Agenzia Delle Entrate)` portal.

Acceptance or Refusal
~~~~~~~~~~~~~~~~~~~~~

.. warning::
   As Odoo does not handle sending signed invoices to :abbr:`PA (Public Administration)` businesses,
   these states cannot be directly triggered by Odoo. When you upload the invoice on the :abbr:`AdE
   (Agenzia Delle Entrate)` portal,Odoo will receive notifications about it, putting the the correct
   :guilabel:`SdI State` on the invoice.

After receiving the invoice through the :abbr:`SdI (Sistema di Interscambio)`, the :abbr:`PA (Public
Administration)` business has 15 days to accept the invoice. If it does, then the process
ends here. If the :abbr:`PA (Public Administration)` business refuses the invoice, it is still
considered valid once it was accepted by the :abbr:`SdI (Sistema di Interscambio)` so you'll have to

> do a credit note to compensate and sent that also to the :abbr:`SdI (Sistema di Interscambio)`.

Reviewer 3: ```suggestion
After receiving the invoice through the :abbr:`SdI (Sistema di Interscambio)`, the :abbr:`PA (Public
Administration)` business has 15 days to accept the invoice. If it does, then the process ends here.
If the :abbr:`PA (Public Administration)` business refuses the invoice, it is still considered valid
once it was accepted by the :abbr:`SdI (Sistema di Interscambio)`. You then have to issue a credit
note to compensate and send it to the :abbr:`SdI (Sistema di Interscambio)`.
```
